pydicom/dataset.py

```diff
@@ -1902,7 +1912,7 @@ def __setattr__(self, name, value):
                 )
                 if config.INVALID_KEYWORD_BEHAVIOR == "WARN":
                     warnings.warn(msg)
-                elif config.INVALID_KEYWORD_BEHAVIOR == "ERROR":
+                elif config.INVALID_KEYWORD_BEHAVIOR == "RAISE":
```

Comment:
Whoops!

Reply:
Yeah, this is why I prefer to use `enum` for this kind of things.